docs/auth/social-auth.md

```diff
@@ -212,13 +212,13 @@ documentation.
 Ensure the "Google" sign-in provider is enabled on the [Firebase Console](https://console.firebase.google.com/project/_/authentication/providers).
 
 Before triggering a sign-in request, you must initialize the Google SDK using your any required scopes and the
-`webClientId`, which can be found on the Firebase Console Settings, as "Web API Key".
+`webClientId`, which can be found in `google-services.json`, under `client/oauth_client/client_id`.
```

Comment:
> "in the `android/app/google-services.json` file as the `client/oauth_client/client_id` property (the id ends with `.apps.googleusercontent.com`)

Probably best to put the entire description here rather than another hint in the code block.